crates/primitives/src/deposit.rs

```diff
@@ -147,12 +147,16 @@ impl DepositInfo {
         })
     }
 
-    fn compute_spend_infos(
+    /// 1. The taproot address computed from the x-only public key in the DRT and the bridge
```

Comment:
This needs a heading:


```suggestion
    /// Computes the witness data for spending the Deposit Request Transaction (DRT) and in the process also validates the following:

    /// 1. The taproot address computed from the x-only public key in the DRT and the bridge
```